.github/workflows/test_iree.yml

```diff
@@ -215,4 +215,10 @@ jobs:
           steps.unet.outcome == 'success' && steps.vae.outcome == 'success'
         run: |
           source ${VENV_DIR}/bin/activate
-          bash iree_tests/benchmarks/benchmark_sdxl_rocm.sh
+          pytest iree_tests/benchmarks/benchmark_sdxl_rocm.py --goldentime 5565
```

Comment:
If you want print statements to appear in the output and plan on running through `pytest`, consider setting `--log-cli-level=info` here and then switching from `print()` to `logging.getLogger().info(...)`. I see that this is also swallowing the output for the benchmark: https://github.com/nod-ai/SHARK-TestSuite/actions/runs/9381948995/job/25832310009?pr=252#step:11:20, making it _required_ to download the results. Ideally results would be streamed to the logs _and_ posted to a summary / uploaded as an artifact.

https://docs.pytest.org/en/stable/how-to/logging.html

For an example of how those fit together, see https://github.com/nod-ai/SHARK-TestSuite/pull/227

Reply:
Here's some example output from IREE's in-tree benchmarks: https://github.com/iree-org/iree/actions/runs/9381093834/job/25849050852#step:9:136

lots of json :P

Reply:
Switched to the log-cli-level flag and logger